package.json

```
@@ -44,7 +44,8 @@
 		"types:preload": "tsc -p src/preload/tsconfig.json",
 		"types:renderer": "tsc -p src/renderer/tsconfig.json",
 		"types": "npm-run-all --print-label --parallel --continue-on-error types:*",
-		"test": "npm-run-all --print-label --parallel lint types",
+		"test": "npm-run-all --print-label --parallel lint types test:vitest",
+		"test:vitest": "vitest --config ./src/renderer/vite.config.js",
```

the `test:vitest` script here will run vitest in watch mode, which I don't think is not desirable if being used as part of the `test` script (especially in the context of CI). I suggest:

- creating two scripts for running vitest: one for watching and the other for just running tests
- update the `test` script to reference the script that only runs the tests

```suggestion
		"test": "npm-run-all --print-label --parallel lint types vitest:run",
		"vitest:run": "vitest run --config ./src/renderer/vite.config.js",
		"vitest:watch": "vitest watch --config ./src/renderer/vite.config.js",		
```

Also updating the development docs to highlight the difference might be helpful

---

https://github.com/digidem/comapeo-desktop/pull/54/commits/abd6bbaa9f036e8ae44f2c1c3595c2c3cf533828
https://github.com/digidem/comapeo-desktop/pull/54/commits/58760ff85e510cb4b37e970429ddf572818f258f